components/directory/details-form.tsx

```
@@ -0,0 +1,495 @@
+  const requiredFieldsCount = 4; // name, link, category, description
+  const completedRequiredFields = ['name', 'link', 'category', 'description'].filter(field => 
+    formData[field] && formData[field].toString().trim()
+  ).length;
+  const progressPercentage = (completedRequiredFields / requiredFieldsCount) * 100;
+
+  return (
+    <div >
+      {/* Enhanced Background Effects */}
+      <div className="absolute inset-0 overflow-hidden pointer-events-none">
+        {/* Gradient orbs matching other sections */}
+        <div className="absolute top-0 -left-4 w-72 h-72 bg-gradient-to-r from-blue-500/10 to-purple-500/10 dark:from-blue-600/20 dark:to-purple-600/20 rounded-full mix-blend-multiply filter blur-xl opacity-70 animate-blob"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)